bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -56,13 +60,21 @@
 		<label>Humidity</label>
 		<description>The Current Humidity Reading of the Thermostat</description>
 		<category>Humidity</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Humidity</tag>
+		</tags>
 		<state readOnly="true" pattern="%d %unit%"/>
 	</channel-type>
 
 	<channel-type id="mode">
 		<item-type>Number</item-type>
 		<label>Mode</label>
 		<description>The Current Operating Mode of the HVAC System</description>
+		<tags>
+			<tag>Control</tag>
```

Comment:
I am not sure if the upcoming `Mode` tag from https://github.com/openhab/openhab-core/pull/4616 would belong here too. I used the tags from the **comfoair** binding as a guide.

Reply:
The `Mode` tag would be the right one as `Climate` is not a sementic tag.
For reference: https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv

Reply:
That's a good question and also relevant for #18315 and #18324. I didn't closely follow openhab/openhab-core#4619, but there might be some hints there.